web/static.go

```diff
@@ -73,7 +76,25 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Cache-Control", "no-cache, max-age=31556926, public")
 
 	staticDir, _ := fileutils.FindDir(model.ClientDir)
-	http.ServeFile(w, r, filepath.Join(staticDir, "root.html"))
+	contents, err := os.ReadFile(filepath.Join(staticDir, "root.html"))
+	if err != nil {
+		// TODO: may be we should return an error page here.
+		return
+	}
+
+	newHTML := string(contents)
+	siteName := c.App.Srv().Config().TeamSettings.SiteName
+	if model.TeamSettingsDefaultSiteName != *siteName {
+		mmTitle := fmt.Sprintf("<title>%s</title>", model.TeamSettingsDefaultSiteName)
+		siteNameTitle := fmt.Sprintf("<title>%s</title>", *siteName)
+		reHTML := regexp.MustCompile(mmTitle)
```

Comment:
IMO it makes no sense to use regexp here at all, a simple `bytes.ReplaceAll` would be easier and more efficient.

Reply:
@mickmister  I'm looking into it. 

> I'm wondering if we can use `og:description` [open graph](https://ogp.me/#optional) data to support the `SiteDescription` variable. Something like this:
> 
> ```go
> siteDescriptionTag := fmt.Sprintf("<meta property="og:description" content="%s" />", *siteDescription)
> ```




> Thanks for this @safakkizkin! I have a few requests to avoid some potential nil pointers, and a few questions about the code. Overall looks awesome 👍
> 
> I'm wondering if we can use `og:description` [open graph](https://ogp.me/#optional) data to support the `SiteDescription` variable. Something like this:
> 
> ```go
> siteDescriptionTag := fmt.Sprintf("<meta property="og:description" content="%s" />", *siteDescription)
> ```
> 
> We would need to make sure we escape any double quotes in the description, to ensure the resulting tag attribute is valid.